src/politics/politics.cpp

```diff
@@ -382,11 +382,60 @@ vector<string> corporateSuffix;
 					 if (stalinmode&&stalinmood < LCSrandom(100) && stalinmood < LCSrandom(100) && stalinmood < LCSrandom(100) && stalinmood < LCSrandom(100)) vote2 = 3;
 					 if (first) switch (lawList[LAW_ELECTIONS])
 					 {
+					 case -2: 

```

Comment:
This ought to be put into a function to avoid duplicate code with house and senate